src/main/java/org/reactome/server/service/controller/citation/CitationController.java

```diff
@@ -0,0 +1,294 @@
+    public ResponseEntity<String> downloadCitation() {
+        String downloadLink = "https://reactome.org/download-data/";
+        return ResponseEntity.ok("\"Name of file\", Reactome, " + generalService.getDBInfo().getVersion() + ", " + downloadLink);
+    }
+
+    // end point for getting data for citing any static citation, given the PMID
+    @GetMapping(value = "/static/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> staticCitation(@ApiParam(value = "PMID of the requested citation", required = true)
+                                                @PathVariable String id) throws Exception {
+        try {
+            HttpGet request = new HttpGet(new URIBuilder()
+                    .setScheme("https")
+                    .setHost("www.ebi.ac.uk")
+                    .setPath("/europepmc/webservices/rest/search")
+                    .setParameter("query", id)
+                    .setParameter("format", "dc")
+                    .setParameter("resultType", "core")
+                    .build());
+
+            try (CloseableHttpClient httpClient = HttpClients.createDefault();
+                 CloseableHttpResponse response = httpClient.execute(request)) {
+
+                // if the EuropePMC API is down or response is not okay for whatever reason...
+                if (response.getStatusLine().getStatusCode() != HttpStatus.OK.value()) {
+                    return new ResponseEntity(STATIC_CITATION_ERROR, HttpStatus.INTERNAL_SERVER_ERROR);
+                }
+
+                else {
+                    String xmlResponse = EntityUtils.toString(response.getEntity(), "UTF-8");
+                    Document doc = DocumentBuilderFactory.newInstance().newDocumentBuilder().parse(new InputSource(new StringReader(xmlResponse)));
+                    return ResponseEntity.ok(doc.getElementsByTagName("dcterms:bibliographicCitation").item(0).getTextContent());
+                }
+
+            }
+            catch (Exception e) {
+                throw new Exception(e);
+            }
+        }
+        catch (Exception e) {
+            infoLogger.error("Exception thrown in staticCitation method", e);
+            return new ResponseEntity(STATIC_CITATION_ERROR, HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    @GetMapping(value = "/export")
+    public void export(@RequestParam Boolean isPathway,
+                                                      @RequestParam String id,
+                                                      @RequestParam String ext,
+                                                      HttpServletRequest request, HttpServletResponse response) throws IOException {
+        Map<String, Object> map;
+        if(isPathway) {
+            map = pathwayCitation(id).getBody();
+        }
+       else {
+            map = getStaticCitationForExport(id);
+        }
+
+        String citationString = "";
+        String filename = "reactome_citation";
```

Comment:
Would be possible to add the `id` or an `stid` so the name becomes unique  ? Hmm, I guess only if it is pathway, isn't it ?